io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/AbstractAnalyseTable.java

```diff
@@ -136,17 +137,16 @@ protected void analyseHeader(Sheet sheet) {
 	protected abstract void headerCell(int num, String text);
 
 	/**
-	 * Analyse the table content.
+	 * Analyse the table content if skipNlines <=0 that don't analyse first row,
+	 * which has already been analyse into the header else analyse starting with
+	 * the skip line
 	 * 
 	 * @param sheet the table sheet
 	 */
 	private void analyseContent(Sheet sheet) {
-		// for each row starting from the second
-		for (int i = 1; i <= sheet.getLastRowNum(); i++) {
+		for (int i = 0; i <= sheet.getLastRowNum(); i++) {
```

Comment:
Why did you change this to start at `0` and not at `skipNlines`? I thought that was the whole point of the changes to `AbstractAnalyseTable`?

Reply:
I feel that we are spinning around the tail.
Perhaps that commit aligns with your thoughts.